src/components/Step.jsx

```diff
@@ -12,11 +12,12 @@ import Transfer from "./Transfer";
 import Connect from "./Connect";
 import Deploy from "./Deploy";
 import Call from "./Call";
+import useKeyFromLocalStore from "../hooks/useKeyFromLocalStore"
 
 const { Text, Paragraph } = Typography;
 
 const Step = ({ stepIndex, prev, next }) => {
-  const [keypair, setKeypair] = useState(null);
+  const [keypair, setKeypair] = useKeyFromLocalStore();
```

Comment:
maybe rename to `useKeypairFromLocalStore` so we keep a distinction between `key` and `keypair`